itests/api/service_accounts_test.py

```diff
@@ -1,69 +1,74 @@
+from __future__ import annotations
+
 from typing import TYPE_CHECKING
 
 from groupy.client import Groupy
 
-from itests.fixtures import api_client, async_api_server  # noqa: F401
 from itests.setup import api_server
-from tests.fixtures import (  # noqa: F401
-    graph,
-    groups,
-    permissions,
-    service_accounts,
-    session,
-    standard_graph,
-    users,
-)
 
 if TYPE_CHECKING:
     from py.local import LocalPath
     from tests.setup import SetupTest
 
 
-def test_get_service_accounts(api_client, users, service_accounts):  # noqa: F811
-    role_users = [username for username, u in users.items() if u.role_user]
-    assert len(role_users) > 0
+def test_get_service_accounts(tmpdir: LocalPath, setup: SetupTest) -> None:
+    with setup.transaction():
+        setup.create_role_user("role@a.co")
+        setup.create_service_account("service@a.co", "team-sre")
```

Comment:
For technically full parity with what was tested before, might as well throw in a `setup.create_user` to be sure non-role users aren't returned as well?

Reply:
Ah, yes, good idea. Will do that.